src/main.js

```diff
@@ -20,13 +19,25 @@ axios.interceptors.response.use(
   },
   error => {
     if (!process.env.VUE_APP_API_KEY) {
-      alert(
-        "Please make a file named '.env.local'. Create a global variable VUE_APP_API_KEY and initialize it with your own key in this file. " +
-          error
-      );
+      console.log(error);
+      router.replace({
+        name: "Error",
+        params: { id: 1 }
```

Comment:
That `1` here looks like a "magic number". It would be better to use some named constant.